__tests__/app.test.js

```diff
@@ -69,8 +69,27 @@ describe("/api/reviews", () => {
     });
 });
 
-describe("/api/reviews/:review_id", () => {
+describe.only("/api/reviews/:review_id", () => {
     test("GET - 200: Responds with review object", () => {
+        return request(app)
+            .get("/api/reviews/1")
+            .expect(200)
+            .then((res) => {
+                const review = res.body.review;
+                expect(review).toMatchObject({
+                    review_id: 1,
+                    title: expect.any(String),
+                    category: expect.any(String),
+                    designer: expect.any(String),
+                    owner: expect.any(String),
+                    review_img_url: expect.any(String),
+                    created_at: expect.any(String),
+                    votes: expect.any(Number),
+                    review_body: expect.any(String),
+                });
+            });
+    });
+    test("GET - 200: Responds with review object containing comment_count", () => {
```

Comment:
I would probably specify the amount of comments we are expecting for this one, as we should know how many comments there are for a specific review